source/chapters/hardware.rst

```diff
@@ -39,21 +20,115 @@ for club, hobby, and radio DJs alike.
              suggestions for what to consider when you are shopping for DJ
              equipment.
 
+Types of DJ Hardware
+====================
+
+Controllers
+-----------
+DJs often use a device called a DJ controller which has knobs, faders, buttons,
+jog wheels, and other components to control DJ software such as Mixxx. It
+is difficult to manipulate Mixxx's controls fast enough while music is playing
+with only a keyboard and mouse while looking at a computer screen. Controllers
+allow quick access to different controls while providing tactile and visual
+feedback indicating the state of each control, such as the position of a knob
+or whether a switch is on or off. This allows the DJ to focus on manipulating
+the music without needing to look at their computer screen all the time.
+Controllers also allow the DJ to use two hands to manipulate two different
+controls at the same time, which is required for many mixing techniques.
+
+DJ controllers typically do not do any actual audio processing. Instead, they
+send signals (typically :term:`MIDI` or :term:`HID` over a USB cable) to the
+computer to instruct DJ software how to manipulate the audio. Many DJ
+controllers include a soundcard with 2 separate stereo outputs built into the
+device. This allows the DJ to transport and setup only one piece of hardware in
+addition to a laptop. Some devices can be used as both controllers and hardware
+mixers.
+
+Soundcards
+----------
+A soundcard or audio interface is a device that allows a computer to send
+output to and receive input from audio equipment. Whether using internal or
+external mixing, it is recommended to use a single soundcard with at least 4
+independent output channels (2 separate stereo pairs). Although the term
+"soundcard" originated with card-shaped devices installed directly into desktop
+computers, most DJs today use USB soundcards with laptop computers.
+
+The headphone jack on most laptops is not a second audio output. Rather,
+plugging headphones into the jack simply redirects the laptop's single stereo
+output from its speakers to your headphones. A splitter cable can be used to
+separate the stereo output of a headphone jack into two separate mono outputs
+for headphone cueing, but it is recommended to use a sound card with at
+least four mono outputs (for two stereo pairs). Such sound cards tend to be
+higher quality than those built into laptops and allow your mix to be enjoyed
+in stereo by your audience.
+
+Unlike some proprietary DJ systems, Mixxx can use any soundcard and any
+:term:`MIDI` or :term:`HID` controller that your :term:`OS <operating system>`
+has drivers to use. If your controller has an integrated soundcard, you may
+choose to use a different soundcard for higher quality audio. Mixxx can also
+use multiple audio devices simultaneously.
+
+Mixers
```

Comment:
Give a hint that there might be an user with hardware volume faders only accessable by the os sound panel.